aflr/api_request.py

```diff
@@ -60,7 +72,9 @@ def _expanded_raise_for_status(self, res):
             res.raise_for_status()
         except HTTPError as e:
             if res.json():
-                raise HTTPError('{} \n Error Message from API: \n {}'.format(str(e), res.json()))
+                raise HTTPError(
+                    "{} \n Error Message from API: \n {}".format(str(e), res.json())
```

Comment:
can you change the order of these 2? because as I user I first check the latest part of the error

Reply:
reordered